packages/google_mobile_ads/android/src/main/java/io/flutter/plugins/googlemobileads/GoogleMobileAdsPlugin.java

```diff
@@ -646,6 +650,150 @@ public void onAdInspectorClosed(@Nullable AdInspectorError adInspectorError) {
           result.success(null);
           break;
         }
+      case "isNativeAdCustomControlsEnabled":
+      {
+        final FlutterAd ad = instanceManager.adForId(call.<Integer>argument("adId"));
+        final FlutterNativeAd nativeAd1 = ad instanceof FlutterNativeAd ? (FlutterNativeAd) ad : null;
+        if (nativeAd1 == null) {
+          result.error(
+                  Constants.ERROR_CODE_UNEXPECTED_AD_TYPE,
+                  "Unexpected ad type for getAdSize: " + ad,
+                  null);
+        } else {
+          result.success(nativeAd1.isCustomControlsEnabled());
+        }
+        break;
+      }
+      case "nativeAdPlaybackPlay":
+      {
+        final FlutterAd ad = instanceManager.adForId(call.<Integer>argument("adId"));
+        final FlutterNativeAd nativeAd1 = ad instanceof FlutterNativeAd ? (FlutterNativeAd) ad : null;
+        if (nativeAd1 == null) {
+          result.error(
+                  Constants.ERROR_CODE_UNEXPECTED_AD_TYPE,
+                  "Unexpected ad type for getAdSize: " + ad,
+                  null);
+        } else {
+          final VideoController videoController = nativeAd1.getVideoController();
+          if (videoController == null) {
+            result.error(
+                    Constants.ERROR_CODE_UNEXPECTED_AD_TYPE,
+                    "Video controller is null",
+                    null);
+          } else {
+            videoController.play();
+            result.success(null);
+          }
+        }
+        break;
+      }
+      case "nativeAdPlaybackPause":
+      {
+        final FlutterAd ad = instanceManager.adForId(call.<Integer>argument("adId"));
+        final FlutterNativeAd nativeAd1 = ad instanceof FlutterNativeAd ? (FlutterNativeAd) ad : null;
+        if (nativeAd1 == null) {
+          result.error(
+                  Constants.ERROR_CODE_UNEXPECTED_AD_TYPE,
+                  "Unexpected ad type for getAdSize: " + ad,
+                  null);
```

Comment:
here and below the first error's description is not updated

Reply:
Updated all of them. Thank you!